lib/setup-chromedriver.sh

```diff
@@ -2,11 +2,14 @@
 
 set -eo pipefail
 
-version=$1
-arch=$2
+VERSION=$1
+ARCH=$2
 
-sudo apt-fast install -y xvfb screen google-chrome-stable
```

Comment:
I'm removing this since it's included by default: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/software-installed-on-github-hosted-runners